### ISSUE TITLE
style_indent Rule doesn't work when new line started after line with comments

### ISSUE THREAD

**Issue by Mishassio** [dalance/svlint#249]
Good case:
![image](https://github.com/dalance/svlint/assets/57057812/2191a060-813f-4822-a5d3-ad07c0659392)

Bad case:
![image](https://github.com/dalance/svlint/assets/57057812/9590f027-b590-481e-8603-dd16eb63cdc9)

**Comment by DaveMcEwan** [comment]
Good catch! That's a good motivation for making another style_indent which works on text, rather than syntax after preprocessing, see #247.
That line is probably being treated as a preprocessor comment (see preproc.rs in sv-parser and Clause 22 of the LRM) so might not be visible to the syntax parser.
